### PR TITLE
feat: support cookie and temp table

### PR DESCRIPTION
**Pull request by youngsofun**
note: 

 temp table test is used to verify cookie works, but logout not supported in this pr, so if temp table is not auto drop at once when session ends, but can still get vaccumed. then it is better for user to still drop it manually

### CODE REVIEW

File: databend_py/connection.py [hunk 1]
```diff
@@ -63,6 +65,23 @@ def get_error(response):
...
+    def __init__(self):
+        super().__init__()
+
+    def set_cookie(self, cookie: Cookie, *args, **kwargs):
+        cookie.domain = ""
+        cookie.path = "/"
+        super().set_cookie(cookie, *args, **kwargs)
+
+    def get_dict(self, domain=None, path=None):
+        # 忽略 domain 和 path 参数，返回所有 Cookie
```
Comment (hantmac) [inline databend_py/connection.py:79]: Plz change the comment to EN.

**Review by hantmac — COMMENTED** [review]
LGTM

**Review by hantmac — APPROVED** [review]

**Comment by youngsofun** [comment]
> databend_py/connection.py

done

File: databend_py/connection.py (hunk 1 above)
Comment (youngsofun) [reply databend_py/connection.py]: fixed

**Comment by youngsofun** [comment]
@hantmac plz return ci and merge

**Comment by youngsofun** [comment]
@hantmac plz return ci and merge

I think GlobleCookieJar already covers the demand in https://github.com/databendlabs/databend-py/pull/33, so the option `persisted_cookies` can be removed later
cc [pengye91](https://github.com/pengye91)

**Pull request merged**